### DISCUSSION TITLE
Is a nice pice of code but you can't sell it

### DISCUSSION THREAD

**Discussion by gotshadowbantempaccount** [nanoexpress/nanoexpress#260] [Ideas] [answered]
## Feature Request
- missing benchmarks on main page ( this guy do it right ) http://gwan.com
**Missing info on main page**
Performance improvement by ~25%
Try to call EXPERIMENTAL_FASTCALL=1 node server.js
- missing information on main page say that is base uWebSockets
- also benchmark link from homepage https://nanoexpress.js.org/benchmark ?? missing where is graphs

A clear and concise description of what the problem is. Ex. I have an issue when [...]

### Describe the solution you'd like

A clear and concise description of what you want to happen. Add any considered drawbacks.

**Comment by dalisoft** [comment, marked as answer]
Hi @gotshadowbantempaccount 

Thank you a lot for notices, i appreciate any PR

**Reply by dalisoft** [reply]
@gotshadowbantempaccount Did you asked for benchmark results? Here it is: https://docs.google.com/spreadsheets/d/1_7vKcAHiTs_O6jLuvUU16l6yO4_hzbT302SHrj-iBsQ/edit?usp=sharing

<img width="662" alt="image" src="https://user-images.githubusercontent.com/3511344/130786076-e0831930-51ca-4905-b96c-199f40de27b7.png">
